src/PhpSpreadsheet/Writer/Xlsx/Worksheet.php

```diff
@@ -1105,7 +1105,7 @@ private function writeCell(XMLWriter $objWriter, PhpspreadsheetWorksheet $pSheet
                     break;
                 case 'f':            // Formula
                     $attributes = $pCell->getFormulaAttributes();
-                    if ($attributes['t'] === 'array') {
+                    if ($attributes != null && $attributes['t'] === 'array') {
```

Comment:
`!==`

Reply:
[Use typed not comparison for $attributes](https://github.com/uofa/phpspreadsheet/pull/1/commits/893419a191ff79127b0a54a8a9f6474715217f16)